contracts/payment/ILemonadeEscrow.sol

```diff
@@ -0,0 +1,68 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
+
+struct RefundPolicy {
+    uint32 timestamp;
+    uint16 percent;
+}
+
+struct Deposit {
+    address token;
+    uint128 amount;
```

Comment:
Why uint128 and not uint256?

Reply:
I thought this would prevent arithmetic overflow when perform calculation with percentage.